pyvista/__init__.py

```diff
@@ -52,7 +52,7 @@
 except KeyError:
     pass
 
-# Grab system flag for auto-closing because of Panel issues
+# Grab system flag for auto-closing
 try:
     # This only sets to false if PYVISTA_AUTO_CLOSE is false
     rcParams['auto_close'] = not os.environ['PYVISTA_AUTO_CLOSE'].lower() == 'false'
```

Comment:
We may be able to drop support for this...

Reply:
Correct, we probably don't need this at the moment, but for now I'm keeping this in as this PR has already changed enough.